doc/source/whatsnew/v0.18.0.txt

```diff
@@ -20,7 +20,9 @@ Check the :ref:`API Changes <whatsnew_0180.api>` and :ref:`deprecations <whatsne
 New features
 ~~~~~~~~~~~~
 
-
+- The Dataframe class got a _repr_latex_ method in order to allow for automatic conversion to latex in
+a ipython/jupyter notebook using nbconvert. Options `display.latex.escape` and `display.latex.longtable`
```

Comment:
use double backticks around ``display.latex.escape`` (and also around ``DataFrame._repr_latex_``)